worlds/dlcquest/Rules.py

```diff
@@ -215,25 +214,35 @@ def has_coin(state, player: int, coins: int):
                  lambda state: state.has("Cut Content Pack", player))
         set_rule(world.get_entrance("Blizzard", player),
                  lambda state: state.has("Season Pass", player))
-        set_rule(world.get_entrance("Boss Door", player),
-                 lambda state: state.has("Big Sword Pack", player) and state.has("Really Big Sword Pack",
-                                                                                 player) and state.has(
-                     "Unfathomable Sword Pack", player))
         set_rule(world.get_location("I Get That Reference!", player),
                  lambda state: state.has("Death of Comedy Pack", player))
         set_rule(world.get_location("Story is Important", player),
                  lambda state: state.has("DLC NPC Pack", player))
         set_rule(world.get_entrance("Pickaxe Hard Cave", player),
                  lambda state: state.has("Pickaxe", player))
+        big_sword_location = world.get_location("Big Sword Pack", player)
+        big_sword_self = lambda state: item_in_locations(state, "Big Sword Pack", player, big_sword_location)
```

Comment:
item_in_locations seems to expect a sequence of location names, and its typing is also wrong, which is a current /main problem I'll look into. But the first problem stands, this shouldn't work.